packages/core/src/lib/errors.ts

```diff
@@ -50,3 +50,8 @@ export class UnauthorizedError extends LatitudeError {
   public statusCode = 401
   public name = 'UnauthorizedError'
 }
+
+export const databaseErrorCodes = {
+  foreignKeyViolation: '23503',
+  uniqueViolation: '23505',
+}
```

Comment:
This should be here https://github.com/latitude-dev/latitude-llm/blob/main/packages/core/src/lib/Transaction.ts#L21

Reply:
will move even if this is just ocd

Reply:
How is this ocd? Is the same unique violation error in two different places. For that matter, put it in the destroy service as a local constant